raiden-ts/tests/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "import/no-unresolved": "off"
```

Comment:
Why that?

Reply:
Even with `eslint-import-resolver-typescript`, I couldn't get `eslint` to detect and respect the `@/` alias in `tests/tsconfig.json`, so in all tests eslint was erroring because it didn't find that package, so I had to disable it.

Reply:
:expressionless: